### PR TITLE
Add foobar2000 2.1.7

### PR DESCRIPTION
**Pull request by icpz**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
From https://www.foobar2000.org/mac:

> It is by no means complete and is not currently under active development

So it’s a beta app that may never leave beta? Not sure it’s a worthy addition.

File: Casks/foobar2000.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'foobar2000' do
+  # note: "2000" is not a version number, but an intrinsic part of the product name
+  version '2.1.7'
+  sha256 '750ab9ebb817a7f4f4dd6b9d17990619037a9b6f829bca8964455ee2c09929c6'
+
+  url "https://www.foobar2000.org/files/264a0d9bf25006ad99fca992b6740779/foobar2000-v#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/foobar2000.rb]: Not the link I see on the website. I’m seeing `https://www.foobar2000.org/getfile/df025b9e7716fdadc2d7a7de36230e38/foobar2000-v2.1.7.dmg`.

**Comment by commitay** [comment]
> So it’s a beta app that may already be abandoned? Not sure it’s a worthy addition.

Agreed.

> Not the link I see on the website

Link seems to change every time I reload the website.

**Comment by vitorgalvao** [comment]
> Link seems to change every time I reload the website.

Even worse! That means we’d require an `url do` block to support an abandoned beta app. Talk about a bad deal.

**Comment by commitay** [comment]
Passing on this because it is unmaintained on mac and it requires a `url do` block.

I'd suggest [creating your own tap.](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md)

**Pull request closed without merging**